app/components/provider-links/template.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if fetchData.isRunning}}
+    {{#content-placeholders as |placeholder|}}
+        <span>
+            <i class="fa-li icon--skeleton">{{placeholder.icon}}</i>
+            {{placeholder.heading subtitle=false}}
+        </span>
+        <div class="provider-links">
+            <div class="provider-links--skeleton">{{placeholder.text lines=2}}</div>
+        </div>
+    {{/content-placeholders}}
+{{else}}
+    <span class="provider-name {{if provider.reviewsWorkflow "moderation-enabled" "moderation-disabled"}}">
+        <i class="fa-li fa {{get iconMap provider.preprintWord}} icon"></i>
+        {{provider.name}}
+    </span>
+    <ul class="provider-links">
+        {{#if provider.reviewsWorkflow}}
+            <li>
+                {{#link-to 'preprints.provider.moderation' provider.id}}
+                    {{t 'global.moderation'}}
+                {{/link-to}}
```

Comment:
FYI, because you took the time to remove the underline on the moderation count, you could get rid of this extended underline:
<img width="135" alt="screen shot 2017-11-14 at 3 12 55 pm" src="https://user-images.githubusercontent.com/6776190/32802588-7d9e7b30-c94e-11e7-8019-0be03136cfb6.png">
by stripping the whitespace inside the `link-to` with [strategic use of tildes](http://handlebarsjs.com/expressions.html#whitespace-control)